libswc/drm.c

```diff
@@ -199,7 +199,7 @@ find_available_crtc(drmModeRes *resources, drmModeConnector *connector, uint32_t
 static bool
 find_available_id(uint32_t *id)
 {
-	int index = ffsl(~drm.taken_ids);
+	int index = __builtin_ffsl(~drm.taken_ids);
```

Comment:
My compiler doesn't have `__builtin_ffsl`. Let me see if I can rewrite this so it doesn't depend on `ffsl`.

Reply:
I pushed a change to just use the CRTC index as the screen ID, so ffsl is no longer used.